### ISSUE TITLE
Pool statistics

### ISSUE THREAD

**Issue by sroze** [josdejong/workerpool#18]
Is there any way to get the number of used and available workers at a given time?

**Comment by josdejong** [comment]
That's an interesting idea. Currently there is no API to get stats, but that would be quite easy to add.

How about something like:

```js
pool.stats()   // {availableWorkers: 3, idleWorkers: 2, queuedTasks: 0}
pool.stats()   // {availableWorkers: 3, idleWorkers: 0, queuedTasks: 35}
```

**Comment by josdejong** [comment]
I've released v2.2.0 containing this new method `pool.stats()`. There is an usage example here: https://github.com/josdejong/workerpool/blob/master/examples/browser/statistics.html

The method returns an object like: 
```js
{
 totalWorkers: 0,
 busyWorkers: 0,
 idleWorkers: 0,
 pendingTasks: 0,
 activeTasks: 0
}
```
